src/main/java/ch/wisv/events/sales/controller/stats/SalesStatsEventController.java

```diff
@@ -0,0 +1,87 @@
+package ch.wisv.events.sales.controller.stats;
+
+import ch.wisv.events.core.exception.normal.EventNotFoundException;
+import ch.wisv.events.core.model.event.Event;
+import ch.wisv.events.core.model.order.Order;
+import ch.wisv.events.core.model.product.Product;
+import ch.wisv.events.core.service.event.EventService;
+import ch.wisv.events.core.service.order.OrderService;
+import ch.wisv.events.sales.service.SalesService;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.List;
+
+/**
+ * SalesScanEventController.
+ */
+@Controller
+@RequestMapping(value = "/sales/stats/event/{key}")
+@PreAuthorize("hasRole('USER')")
+public class SalesStatsEventController {
+    /** Default return redirect on error. */
+    private static final String ERROR_REDIRECT = "redirect:/sales/stats/";
+
+    /** OrderService. */
```

Comment:
You're not making anyone happy with comments like this... Just remove 'm if they don't add anything

Reply:
done